docs/content/preview/explore/query-1-performance/pg-stat-statements.md

```diff
@@ -142,7 +141,105 @@ Time Span (ms) | Resolution (ms)
 209715.2-419430.4 (3.49-6.99 min) | 26214.4 (~26s)
 419430.4-838860.8 (6.99-13.98 min) | 52428.8 (~52s)
 838860.8-1677721.6 (13.98-27.96 min) | 104857.6 (~104s)
--->
+
+You can change the number of buckets using the `yb_hdr_bucket_factor` parameter. Valid values are 8, 16 (default), or 32. For example:
+
+```sh
+pg_stat_statements.yb_hdr_bucket_factor = 32
+```
+
+This changes the buckets as shown in the following table.
+
+Time Span (ms) | Resolution (ms)
+| :--- | :--- |
+| 0-3.2 | 0.1
+3.2-6.4 | 0.2
+6.4-12.8 | 0.4
+12.8-25.6 | 0.8
+25.6-51.2 | 1.6
+51.2-102.4 | 3.2
+102.4-204.8 | 6.4
+204.8-409.6 | 12.8
+409.6-819.2 | 25.6
+819.2-1638.4 | 51.2
+... |
+209715.2-419430.4 (3.49-6.99 min) | 13107.2
+419430.4-838860.8 (6.99-13.98 min) | 26214.4
+838860.8-1677721.6 (13.98-27.96 min) | 52428.8
+
+### Show latency percentiles
+
+Use the `yb_get_percentile` function to show percentiles for a given histogram. This function takes a JSONB histogram (including `yb_latency_histogram`) and a target percentile (as an integer) as input.
+
+```sql
+\df yb_get_percentile
+```
+
+```output
+                                         List of functions
+   Schema   |       Name        | Result data type |           Argument data types           | Type
+----------- +-------------------+------------------+-----------------------------------------+------
+ pg_catalog | yb_get_percentile | double precision | hist jsonb, percentile double precision | func
+(1 row)
+```
+
+The following examples show how to display various percentile latencies for a given JSONB histogram.
+
+```sql
+SELECT yb_get_percentile('[{"[0.1,0.2)": 4}, {"[0.2,0.3)": 1}, {"[0.8,0.9)": 1}, {"[1.1,1.2)": 3}, {"[1.5,1.6)": 1}, {"[2.0,2.2)": 4}, {"[3.2,3.6)": 1}]', 99);
+```
+
+```output
+ yb_get_percentile 
+-------------------
+               3.6
+(1 row)
+```
+
+```sql
+SELECT yb_get_percentile('[{"[0.1,0.2)": 4}, {"[0.2,0.3)": 1}, {"[0.8,0.9)": 1}, {"[1.1,1.2)": 3}, {"[1.5,1.6)": 1}, {"[2.0,2.2)": 4}, {"[3.2,3.6)": 1}]', 90);
+```
+
+```output
+ yb_get_percentile 
+-------------------
+               2.2
+(1 row)
+```
+
+```sql
+SELECT yb_get_percentile('[{"[0.1,0.2)": 4}, {"[0.2,0.3)": 1}, {"[0.8,0.9)": 1}, {"[1.1,1.2)": 3}, {"[1.5,1.6)": 1}, {"[2.0,2.2)": 4}, {"[3.2,3.6)": 1}]', 50);
+```
+
+```output
+ yb_get_percentile 
+-------------------
+               1.2
+(1 row)
+```
+
+```sql
+SELECT query, calls, total_time, min_time, max_time, mean_time, rows, 
```

Comment:
Can we put a description similar to following: 
Display the P99, P95 and P90  latency by augmenting the pg_stat_statements output for specific query.